view/page_premium.go

```diff
@@ -0,0 +1,62 @@
+package view
+
+import (
+	"fmt"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/kaytu-io/kaytu/controller"
+	"github.com/kaytu-io/kaytu/pkg/style"
+	"github.com/kaytu-io/kaytu/pkg/utils"
+	"github.com/kaytu-io/kaytu/view/responsive"
+	"strings"
+)
+
+type PremiumPage struct {
+	helpController *controller.Help
+
+	responsive.ResponsiveView
+}
+
+func NewPremiumPage(helpController *controller.Help) PremiumPage {
+	return PremiumPage{
+		helpController: helpController,
+	}
+}
+
+func (m PremiumPage) OnClose() Page {
+	return m
+}
+func (m PremiumPage) OnOpen() Page {
+	m.helpController.SetKeyMap([]string{
+		"ctrl+c: exit",
+	})
+	return m
+}
+
+func (m PremiumPage) Init() tea.Cmd { return nil }
+
+func (m PremiumPage) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	return m, nil
+}
+
+func (m PremiumPage) View() string {
+	message := []string{fmt.Sprintf("You have reached the limit for this user and organization.\n"+
+		"You need to purchase for premium user or organization to use unlimitted edition, contact us:\n"+
```

Comment:
Fix this message as well.
Put all the messages in same place so that it's easier to change in single place.